peps/pep-0007.rst

```diff
@@ -26,12 +26,21 @@ particular rule:
    clean up someone else's mess (in true XP style).
 
 
-C dialect
-=========
+C standards
+===========
+
+Follow the following standards.
+For features that aren't in the relevant standard (like atomics or some C11
+features in public headers), use CPython-specific wrappers.
```

Comment:
I had to follow the wikipedia link to optional features below to understand this. The point is that certain features are declared optional by the standard, and for those, if we want to use them, we must use CPython-specific headers. (But why not allow checking the standard's feature-check macros directly, if the main code just has to have two versions? Wrappers are useful if you'd be checking for the feature-check macros repeatedly (like with atomics) or if a shorthand macro could help not seeing the feature checks at all in the rest of the code.